src/server/config/schema.js

```diff
@@ -30,9 +30,9 @@ module.exports = () => Joi.object({
     exclusive: Joi.boolean().default(false)
   }).default(),
 
-  uuid: Joi.string().guid().default(),
 
   server: Joi.object({
+    uuid: Joi.string().guid().default(),
```

Comment:
Out of curriosity, is this change backwards compatible? Did we release a version that had uuid at the root of the config?